charts/prometheus-kafka-exporter/values.yaml

```
@@ -71,6 +71,9 @@ service:
   port: 9308
   labels: {}
   annotations: {}
+  # Specifies a custom nodePort for external use.
+  # Must be used with service.type=NodePort.
+  # nodePort: 30100
```

Thank you, @KimJoonSeo, for your PR. I'd suggest not to comment the field out.

```suggestion
  nodePort: null
```